src/helpers/entityValue.ts

```diff
@@ -1,16 +1,39 @@
+import { jsonRpcRequest } from './utils';
+
 type Vote = {
   vp_by_strategy: number[];
 };
 
 type Proposal = {
-  vp_value_by_strategy: number[];
+  network: string;
+  strategies: any[];
+  start: number;
 };
 
+const OVERLORD_URL = process.env.OVERLORD_URL ?? 'https://overlord.snapshot.org';
+// Round strategy values to 9 decimal places
+const STRATEGIES_VALUE_PRECISION = 9;
+
+export async function getStrategiesValue(proposal: Proposal): Promise<number[]> {
```

Comment:
Ideally we should keep same naming everywhere, here I would just call it  `getVpValueByStrategy`